reduction/fit_pointing_model.py

```diff
@@ -49,26 +49,26 @@
     raise RuntimeError('Please specify a single CSV data file as argument to the script')
 filename = args[0]
 
-# Set up logging: logging everything (DEBUG & above)
-logging.basicConfig(level=logging.DEBUG, stream=sys.stdout, format="%(levelname)s: %(message)s")
-logger = logging.root
+# Set up logging
+logging.basicConfig(level=logging.WARNING, stream=sys.stdout, format="%(levelname)s: %(message)s")
+logger = logging.getLogger('fit_pointing_model')
 logger.setLevel(logging.DEBUG)
 
 # Load old pointing model, if given
 old_model = None
 if opts.pmfilename:
     try:
         old_model = katpoint.PointingModel(open(opts.pmfilename).readline())
-        logger.debug("Loaded %d-parameter pointing model from '%s'" % (len(old_model), opts.pmfilename))
+        logger.debug("Loaded %d-parameter pointing model from '%s'", len(old_model), opts.pmfilename)
     except IOError:
-        logger.warning("Could not load old pointing model from '%s'" % (opts.pmfilename,))
+        logger.warning("Could not load old pointing model from '%s'", opts.pmfilename)
 
 # Load data file in one shot as an array of strings
-data = np.loadtxt(filename, dtype=np.str, comments='#', delimiter=', ')
+data = np.loadtxt(filename, dtype=str, comments='#', delimiter=', ')
```

Comment:
Yes, that's a good one! 😂

It is there because that's what `analyse_point_source_scans` produces.

If I change it to `delimiter=','`, the script crashes due to the whitespace. An alternative that can safely handle both comma and comma-space cases is:
```Python
data = np.loadtxt(filename, dtype=str, comments='#', delimiter=',')
data = np.char.strip(data)
```

Maybe I should add this, since Henk might soon feed this script optical pointing data that was not produced by a.p.s.s...